tests/test_requirements.txt

```diff
@@ -1,3 +1,5 @@
+requests
+pytz
```

Comment:
Where do you use it?

Reply:
requests is used in clickhouse_connect/driver/httpclient and pytz is used in a couple of places, include clickhouse_connect/datatypes/temporal.  They used to get pulled in to the test environment as I side effect of installing clickhouse_connect using pip (pip install -e .), but pip didn't like the "developer_only" version. 

So instead I don't install the project at all (which actually is cleaner) and install all the test and lint requirements explicitly (as well as compiling the cython modules explicitly as well).